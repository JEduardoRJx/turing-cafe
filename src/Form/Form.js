import React from 'react'
import './Form.scss'

const Form = () => {
  return(
    <form>
      <input type='text' placeholder='Name' />
      <input type='text' placeholder='Date (MM/DD)' />
      <input type='text' placeholder='Time' />
      <input type='text' placeholder='Number of Guests' />
      <button type='button'>Make Reservation</button>
    </form>
  )
}

export default Form;