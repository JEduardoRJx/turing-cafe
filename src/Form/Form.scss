form {
  // background-color: blue;
  width: 100%;
  height: 30px;
  display: flex;
  justify-content: space-around;
  margin: 30px 0px;
}

input {
  height: 30px;
  border: none;
  width: 200px;
  text-align: center;
  font-size: 1em;
  border-radius: 5px;
}

button {
  height: 30px;
  border: none;
  width: 200px;
  font-size: 1em;
  border-radius: 5px;
  background-color: #D2AA81;
  font-weight: bold;
}