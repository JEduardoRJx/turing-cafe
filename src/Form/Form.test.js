import React from 'react';
import Form from './Form';
import { shallow } from 'enzyme'

describe('Form', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Form />)
  });

  it('should create a snapshot of Form', () => {
    expect(wrapper).toMatchSnapshot();
  });

})