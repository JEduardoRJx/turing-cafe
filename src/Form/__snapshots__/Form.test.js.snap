// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should create a snapshot of Form 1`] = `
<form>
  <input
    name="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
  />
  <input
    name="date"
    onChange={[Function]}
    placeholder="Date (MM/DD)"
    type="text"
  />
  <input
    name="time"
    onChange={[Function]}
    placeholder="Time"
    type="text"
  />
  <input
    name="numGuests"
    onChange={[Function]}
    placeholder="Number of Guests"
    type="text"
  />
  <button
    type="button"
  >
    Make Reservation
  </button>
</form>
`;
